saleor/graphql/product/tests/test_product.py

```diff
@@ -9748,6 +9751,185 @@ def test_update_or_create_variant_stocks_empty_stocks_data(variant, warehouses):
     assert stock.quantity == 5
 
 
+@patch("saleor.plugins.manager.PluginsManager.product_variant_back_in_stock")
+@patch("saleor.plugins.manager.PluginsManager.product_variant_out_of_stock")
+def test_update_or_create_variant_with_back_in_stock_webhooks_only(
+    product_variant_stock_out_of_stock_webhook,
+    product_variant_back_in_stock_webhook,
+    settings,
+    variant,
+    warehouses,
+    info,
+    shipping_zone,
+):
+
+    for warehouse in warehouses:
+        warehouse.shipping_zones.set([shipping_zone])
+        warehouse.save()
```

Comment:
IMHO `.save()` is not needed here 🤔

Reply:
Make sure that you apply this comment to all tests where you call `.save` and `.set`